### PR TITLE
Split test development to be parallelized

### LINKED ISSUE #2866: Parallelize running the chromatica tests
**Is your feature request related to a problem? Please describe.**
We should parallelize as much of the simorgh pipeline as possible

**Describe the solution you'd like**
Move running of chromatica to a separate build step that runs in parallel with everything else.

**Describe alternatives you've considered**
Democracy is non negotiable

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by mbzain76**
Resolves https://github.com/bbc/simorgh/issues/2866

**Overall change:** _A very high-level summary of easily-reproducible changes that can be Parallelize development tests.

**Code changes:**
Split related code in make file.
Add function runDevelopmentChromaticTests.
Add new stage "Chromatic Test Development".

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -20,6 +20,11 @@ def runDevelopmentTests(){
   sh 'make developmentTests'
 }
 
+def runDevelopmentChromaticTests(){
+  sh 'make install'
```
Comment (chris-hinds) [inline Jenkinsfile:24]: @mbzain76 do we need to run the `make install` step in these 3 methods? running this step prior just once will save us some time.

**Review by chris-hinds — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (chris-hinds) [reply Jenkinsfile:24]: we could move the `make install` command to just above these two parallel steps.

Comment (mbzain76) [reply Jenkinsfile:24]: `make install` should be added to every stage, it was failing without it.

**Comment by chris-hinds** [comment]
Closing this PR as it will b e completed as part of a refactor to the pipeline.

**Pull request closed without merging**